server/supernova/sc/sc_ugen_factory.cpp

```diff
@@ -67,6 +67,7 @@ Unit* sc_ugen_def::construct(sc_synthdef::unit_spec_t const& unit_spec, sc_synth
 
     /* initialize members from synth */
     unit->mParent = static_cast<Graph*>(s);
+    unit->mParentIndex = index;
```

Comment:
I don't see a reference to this member anywhere else in this PR, so for my own curiosity - is this a new member of Unit?

Reply:
`mParentIndex` is the index of the Ugen within the Synth. It has always been a member of Unit but Supernova simply forgot about it.

Reply:
ok - I was concerned with breaking the UGen interface. Thanks!